phpBB/config/installer/container/services_install_data.yml

```diff
@@ -0,0 +1,44 @@
+services:
+    installer.install_data.add_bots:
+        class: phpbb\install\module\install_data\task\add_bots
+        arguments:
+            - @installer.helper.config
+            - @installer.helper.iohandler
+            - @installer.helper.container_factory
+            - @language
+            - %core.root_path%
+            - %core.php_ext%
+        tags:
+            - { name: install_data_install, order: 2 }
```

Comment:
maybe we could use 10/20/30.... to be make our life easier if we want to add a new task later